juniper_rocket/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # [master]
 
+### Rocket updated to v0.4
```

Comment:
I placed this under `# [master]` because the version didn't change, so it's still on the `master` branch of the repo. Let me know if you'd like this to be different.

Reply:
Nope, that works perfect!